doc/lua_api.txt

```diff
@@ -6420,8 +6420,8 @@ object you are working with still exists.
           nodes square)
     * `selected_mode` is the mode index to be selected after modes have been changed
     (0 is the first mode).
-* `set_sky(parameters)`
-    * `parameters` is a table with the following optional fields:
+* `set_sky(skybox_parameters)`
```

Comment:
'parameters' is better because these parameters contain several things that are unrelated to 'skybox'. Also because 'skybox' is used in the parameters to mean one particular sky mode.

Reply:
Perhaps 'sky_parameters'?

Reply:
Ah, i see we have:
`SkyboxParams skybox_params = player->getSkyParams();`
in code, which might be why you chose this, but that line also has `getSkyParams`, so 'sky_parameters' is suitable.

Reply:
To be clear, i am not suggesting altering `skybox_params` in the code, i am suggesting using `sky_parameters` in docs and code comments only.

Reply:
Done, thanks.